roles/acme_certificate/meta/argument_specs.yml

```diff
@@ -87,6 +87,7 @@ argument_specs:
           - ns1
           - inwx
           - ansibletest
+          - hetzner
         description:
           - Determines which DNS provider is used for DNS challenges.
           - Every DNS provider requires further configuration.
```

Comment:
Can you add:
```suggestion
          - Every DNS provider requires further configuration.
          - The value C(hetzner) is allowed since felixfontein.acme 0.7.0.
```